librz/core/cmd_descs/cmd_meta.yaml

```diff
@@ -0,0 +1,407 @@
+      - name: Cf-
+        cname: meta_format_remove
+        summary: Remove format string from the current address
+        args: []
+      - name: Cf-*
+        cname: meta_format_remove_all
+        summary: Remove all format string marks
+        args: []
+  - name: Cd
+    summary: Manage the raw data metainformation
+    subcommands:
+      - name: Cd
+        cname: meta_data
+        summary: Set the "data" mark to the current address / List all "data" marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: repeat
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: Cd.
+        cname: meta_data_at
+        summary: Show the data mark at the current offset
+        args: []
+      - name: Cd-
+        cname: meta_data_remove
+        summary: Remove the data mark from the current address
+        args: []
+      - name: Cd-*
+        cname: meta_data_remove_all
+        summary: Remove all data marks
+        args: []
+  - name: Ch
+    summary: Manage the "hidden" mark metainformation
+    subcommands:
+      - name: Ch
+        cname: meta_hidden
+        summary: Set the "hidden" mark to the current address / List all "hidden" marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: Ch-
+        cname: meta_hidden_remove
+        summary: Remove the "hidden" mark from the current address
+        args: []
+      - name: Ch-*
+        cname: meta_hidden_remove_all
+        summary: Remove all "hidden" marks
+        args: []
+  - name: Cm
+    summary: Manage the "magic" mark metainformation
+    subcommands:
+      - name: Cm
+        cname: meta_magic
+        summary: Set the magic to the current address / List all magic marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: Cm-
+        cname: meta_magic_remove
+        summary: Remove the magic mark from the current address
+        args: []
+      - name: Cm-*
+        cname: meta_magic_remove_all
+        summary: Remove all magic marks
+        args: []
+  - name: Cr
+    summary: Manage the "mun" mark metainformation
+    subcommands:
+      - name: Cr
+        cname: meta_run
+        summary: Set the magic to the current address / List all magic marks
```

Comment:
`s/magic/somethingelse/g`? this is probably a copy-paste error

Reply:
Yeah, it's "run", will fix that